README.md

```diff
@@ -110,6 +149,32 @@ model and controller modules already exist, use the following command:
 $ mix torch.gen.html Blog Post posts --no-schema --no-context --web Admin title:string body:text published_at:datetime published:boolean views:integer
 ```
 
+Would only generate these files:
+
+```bash
+* creating priv/templates/phx.gen.html/edit.html.heex
+* creating priv/templates/phx.gen.html/form.html.heex
+* creating priv/templates/phx.gen.html/index.html.heex
+* creating priv/templates/phx.gen.html/new.html.heex
+* creating priv/templates/phx.gen.html/show.html.heex
+* creating priv/templates/phx.gen.html/controller_test.exs
+* creating priv/templates/phx.gen.html/controller.ex
+* creating priv/templates/phx.gen.html/view.ex
+* creating priv/templates/phx.gen.context/access_no_schema.ex
+* creating priv/templates/phx.gen.context/context.ex
+* creating priv/templates/phx.gen.context/schema_access.ex
+* creating priv/templates/phx.gen.context/test_cases.exs
+* creating priv/templates/phx.gen.context/context_test.exs
+* creating lib/phx1_6_web/controllers/admin/post_controller.ex
+* creating lib/phx1_6_web/templates/admin/post/edit.html.heex
+* creating lib/phx1_6_web/templates/admin/post/form.html.heex
+* creating lib/phx1_6_web/templates/admin/post/index.html.heex
+* creating lib/phx1_6_web/templates/admin/post/new.html.heex
+* creating lib/phx1_6_web/templates/admin/post/show.html.heex
+* creating lib/phx1_6_web/views/admin/post_view.ex
+* creating test/phx1_6_web/controllers/admin/post_controller_test.exs
```

Comment:
I'm not sure this is valuable, this is the same list as above but without the migrations.

Reply:
Same comment here as above.  I'm not really keen on keeping a full list of files as the actual files and names probably are mostly unimportant to the end user in the context of reading this readme.